merlin/models/tf/transformers/transformer_block.py

```diff
@@ -0,0 +1,548 @@
+#
```

Comment:
Renaming this file to `block.py` is probably more in-line with the rest of our naming